### ISSUE TITLE
[BUG] CANNOT LOG IN IN YOUTUBE

### ISSUE THREAD

**Issue by djlivooo** [Moosync/Moosync#311]
**Describe the bug**
I have performed the steps given in the wiki. I created the app on the Google Developers page. Copy and paste the Client ID and Secret. And when I hit connect, it goes fine, no problem, enter my email that I added to the app. But it does not connect, nothing comes out.

**To Reproduce**
Just hover over the person icon in the top right corner and click on the YOUTUBE button. Wait for the browser to open the google accept, and says that is done, but in the app doesn't appear my account.

**Expected behavior**
Get my account synced in Moonsync.

- OS: WINDOWS
- Version Windows 11 Home Single Language 22H2

**Comment by Ovenoboyo** [comment]
Does the popup in Moosync disappear after the Google login succeeds?

**Comment by djlivooo** [comment]
Yeah. It says that it succeed, and then redirect me to the app, but nothing happens. I look again to the icon in the app, and in the YouTube logo says "Connect"

**Comment by Ovenoboyo** [comment]
Could you check if something shows up in settings > logs when you try to login?

**Comment by djlivooo** [comment]
![imagen](https://github.com/Moosync/Moosync/assets/98977517/0fbc3a70-9bf4-4c6f-9bbc-ff4d9b81fec5)
It shows this

**Comment by Ovenoboyo** [comment]
401 usually happens when you're client ID or secret is invalid. Could you double check those values? Also could you check if there are no spaces before or after those values

**Comment by djlivooo** [comment]
> 401 usually happens when your client ID or secret is invalid. Could you double check those values? Also could you check if there are no spaces before or after those values

Ok, I'll check.

**Comment by djlivooo** [comment]
> 401 usually happens when your client ID or secret is invalid. Could you double check those values? Also could you check if there are no spaces before or after those values

Ok I think it was a problem with the clients.
![imagen](https://github.com/Moosync/Moosync/assets/98977517/86ad4a55-85b5-426e-96ad-8e2e626ab4c5)

Now it appears my username. Thanks for the help.
